src/core/opamStd.ml

```diff
@@ -1137,11 +1137,8 @@ module OpamSys = struct
         let rec f a =
           match input_line c with
           | x ->
-            (* Treat MSYS2's variant of `cygwin1.dll` called `msys-2.0.dll` equivalently.
-               Confer https://www.msys2.org/wiki/How-does-MSYS2-differ-from-Cygwin/ *)
```

Comment:
Good catch - perhaps this comment (the reference to MSYS2's wiki is useful) could be down below in the `is_cygwin_variant` function?

Reply:
Yep!